src/index.html

```diff
@@ -11,6 +11,46 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      action="index.html" 
+      method="post" 
+      data-qa="big" 
+      class="form form--big"
```

Comment:
```suggestion
      class="form"
```
as there only 2 combinations we may consider big as default and get rid of additional selectors
but not critical